src/puppetlabs/puppetdb/cli/benchmark.clj

```diff
@@ -284,6 +290,8 @@
    ["-F" "--facts FACTS" "Path to a directory containing sample JSON facts (files must end with .json)"]
    ["-C" "--catalogs CATALOGS" "Path to a directory containing sample JSON catalogs (files must end with .json)"]
    ["-R" "--reports REPORTS" "Path to a directory containing sample JSON reports (files must end with .json)"]
+   ["-A" "--archive ARCHIVE" "Path to a PuppetDB export tarball. Incompatible with -C, -F, or -R, or -D"]
+   ["-D" "--default" "Use the sample data in acceptance/benchmark. Incompatible with -C, -F, -R, or -A"]
```

Comment:
Depending on the outcome of "where do we store it", could probably just change it to "Use sample seed data" or something similar. This looks to me like it only supports facts. We should probably state that, and out of curiosity, why doesn't it work with reports or catalogs since it only applies to facts?